manifest.md

```diff
@@ -191,6 +191,10 @@ Unlike the [Manifest List](#manifest-list), which contains information about a s
 
         This REQUIRED property is the digest of the content, as defined by the [Descriptor](descriptor.md) digest format.
 
+    - **`urls`** *array*
+
+       For an ordinary layer, this is empty, and the layer contents can be retrieved directly from the registry. For a layer with mediatype of application/vnd.docker.image.rootfs.foreign.diff.tar.gzip, this contains a non-empty list of URLs from which this object can be downloaded.
```

Comment:
The `application/vnd.docker.image.rootfs.foreign.diff.tar.gzip` media type is not defined in this spec, so we probably want to either define it or remove the restriction.  Is there a reason for the media-type restriction or for making this layer-specific?  I think “when I wrote this, you could get the blob from {urls}” is a useful information for descriptor-targets in general (in which case this field should go [here][1]).

[1]: https://github.com/opencontainers/image-spec/blob/master/descriptor.md#properties